cpp/oneapi/dal/util/load_graph.hpp

```diff
@@ -33,5 +33,6 @@ template <typename Descriptor>
 using output_type = typename Descriptor::output_type;
 
 template <typename Descriptor = load_graph::descriptor<>, typename DataSource = csv_data_source>
-output_type<Descriptor> load(const Descriptor &desc, const DataSource &data_source);
+ONEAPI_DAL_EXPORT output_type<Descriptor> load(const Descriptor &desc,
```

Comment:
why is it still in utils?
Why the previous declaration via function template was removed?

Reply:
ONEAPI_DAL_EXPORT was added. 
The file will be moved from utils

Reply:
waiting answer from dal team.

Reply:
changed for header based impl.